### ISSUE TITLE
mtex 5.5.0 crashes on Matlab 2020b

### ISSUE THREAD

**Issue by eddahbi-fis** [mtex-toolbox/mtex#610]
I tried following the steps in the installation section at https://mtex-toolbox.github.io/download  . However, the problem is that once I type 

```matlab

startup_mtex

```
 in the command window, I get the error seen in the attached photo. I am running Matlab 2020b. I am able to run mtex 5.4.0 without any trouble. But I face this problem when running mtex 5.5.0 . I would really appreciate it if you could help me to solve this issue. 

![error_matlab_mtex5 5](https://user-images.githubusercontent.com/75380911/100894488-d5c6be80-34bc-11eb-9e3d-3837deb3a890.PNG)

**Comment by kilir** [comment]
Could you let us know what operation system you have running and what the report says?
Any other error message?
You could try to remove the content of extern/nfft or extern/nfft:openMP and see if this helps. If so, you may have to build those yourselves. See here: https://github.com/mtex-toolbox/mtex/issues/548

Cheers,
Rüdiger

**Comment by eddahbi-fis** [comment]
I am using Windows 10 Enterprise (64-bit).  This is what I get in the command window when I try to install mtex 5.5

```

initialize MTEX 5.5.0  ....
--------------------------------------------------------------------------------
           Illegal instruction detected at 2020-12-02 16:40:29 +0100
--------------------------------------------------------------------------------

Configuration:
  Crash Decoding           : Disabled - No sandbox or build area path
  Crash Mode               : continue (default)
  Default Encoding         : windows-1252
  Deployed                 : false
  Graphics Driver          : Unknown hardware 
  Graphics card 1          : NVIDIA ( 0x10de ) NVIDIA GeForce 210  Version 21.21.13.4201 (2016-11-14)
  Java Version             : Java 1.8.0_202-b08 with Oracle Corporation Java HotSpot(TM) 64-Bit Server VM mixed mode
  MATLAB Architecture      : win64
  MATLAB Entitlement ID    : 3192465
  MATLAB Root              : C:\Program Files\MATLAB\R2020b
  MATLAB Version           : 9.9.0.1524771 (R2020b) Update 2
  OpenGL                   : hardware
  Operating System         : Microsoft Windows 10 Enterprise
  Process ID               : 5136
  Processor ID             : x86 Family 6 Model 58 Stepping 9, GenuineIntel
  Session Key              : fa11bc8c-61ea-4d26-b6dd-59fab3a2d90b
  Window System            : Version 10.0 (Build 17763)

Fault Count: 1


Abnormal termination:
Illegal instruction

Current Thread: 'MCR 0 interpreter thread' id 15236

Register State (from fault):
  RAX = 0000000000000000  RBX = 00000200f439c050
  RCX = 00000200f439c040  RDX = 0000000000000010
  RSP = 000000049aef6370  RBP = 000000049aef66e0
  RSI = 3ff0000000000000  RDI = 0000000000000001
 
   R8 = 00000200f439c050   R9 = 00000200f439c0d0
  R10 = 0000000000000011  R11 = 0000000000000090
  R12 = 000000000000000c  R13 = 000000049aef73b8
  R14 = 000000049aef65e8  R15 = 000000049aef66e0
 
  RIP = 0000000067600f64  EFL = 00010202
 
   CS = 0033   FS = 0053   GS = 002b

Stack Trace (from fault):
[  0] 0x0000000067600f64 C:\Users\meddahbi\Desktop\MTEX-Moh\mtex-5.5.0\extern\nfft_openMP\nfsftmex.mexw64+00266084 alpha_al_all+00001892
[  1] 0x0000000067604020 C:\Users\meddahbi\Desktop\MTEX-Moh\mtex-5.5.0\extern\nfft_openMP\nfsftmex.mexw64+00278560 nfsft_precompute+00000352
[  2] 0x00000000675c1df6 C:\Users\meddahbi\Desktop\MTEX-Moh\mtex-5.5.0\extern\nfft_openMP\nfsftmex.mexw64+00007670 mexFunction+00002358
[  3] 0x00007fff8233a289 C:\Program Files\MATLAB\R2020b\bin\win64\libmex.dll+00565897 mexfile::Inspector::needs_upgrade+00015977
[  4] 0x00007fff8233b67b C:\Program Files\MATLAB\R2020b\bin\win64\libmex.dll+00571003 MexRetrieveVersion+00003579
[  5] 0x00007fff8233b7ec C:\Program Files\MATLAB\R2020b\bin\win64\libmex.dll+00571372 MexRetrieveVersion+00003948
[  6] 0x00007fff8233b9b4 C:\Program Files\MATLAB\R2020b\bin\win64\libmex.dll+00571828 MexRetrieveVersion+00004404
[  7] 0x00007fff8231c209 C:\Program Files\MATLAB\R2020b\bin\win64\libmex.dll+00442889 mexUnlock_800+00023993
[  8] 0x00007fff8a3626c1                     bin\win64\pgo\m_dispatcher.dll+00665281 Mfh_file::dispatch_file_common+00000165
[  9] 0x00007fff8a361d62                     bin\win64\pgo\m_dispatcher.dll+00662882 Mfh_error::lookup+00006498
[ 10] 0x00007fff8a362615                     bin\win64\pgo\m_dispatcher.dll+00665109 Mfh_file::dispatch+00000045
[ 11] 0x00007fff76c0875f                            bin\win64\pgo\m_lxe.dll+00558943 CGGBuiltinImpl+00183151
[ 12] 0x00007fff76c07e37                            bin\win64\pgo\m_lxe.dll+00556599 CGGBuiltinImpl+00180807
[ 13] 0x00007fff76beb484                            bin\win64\pgo\m_lxe.dll+00439428 CGGBuiltinImpl+00063636
[ 14] 0x00007fff76d2bb61                            bin\win64\pgo\m_lxe.dll+01751905 MathWorks::lxe::ShutdownLxeEngine+00033653
[ 15] 0x00007fff76bef20d                            bin\win64\pgo\m_lxe.dll+00455181 CGGBuiltinImpl+00079389
[ 16] 0x00007fff76bee4dd                            bin\win64\pgo\m_lxe.dll+00451805 CGGBuiltinImpl+00076013
[ 17] 0x00007fff76bdd187                            bin\win64\pgo\m_lxe.dll+00381319 CGGBuiltinImpl+00005527
[ 18] 0x00007fff76bdcd8f                            bin\win64\pgo\m_lxe.dll+00380303 CGGBuiltinImpl+00004511
[ 19] 0x00007fff76be8b2c                            bin\win64\pgo\m_lxe.dll+00428844 CGGBuiltinImpl+00053052
[ 20] 0x00007fff76bed185                            bin\win64\pgo\m_lxe.dll+00446853 CGGBuiltinImpl+00071061
[ 21] 0x00007fff76bec19c                            bin\win64\pgo\m_lxe.dll+00442780 CGGBuiltinImpl+00066988
[ 22] 0x00007fff76d02c1f                            bin\win64\pgo\m_lxe.dll+01584159 mwboost::serialization::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >+00630415
[ 23] 0x00007fff76d02b07                            bin\win64\pgo\m_lxe.dll+01583879 mwboost::serialization::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >+00630135
[ 24] 0x00007fff76d02a16                            bin\win64\pgo\m_lxe.dll+01583638 mwboost::serialization::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >::singleton<mwboost::serialization::extended_type_info_typeid<MathWorks::lxe::PreLineExecutionEvent> >+00629894
[ 25] 0x00007fff76d42d7c                            bin\win64\pgo\m_lxe.dll+01846652 mwboost::archive::detail::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>+00024116
[ 26] 0x00007fff76d42d2a                            bin\win64\pgo\m_lxe.dll+01846570 mwboost::archive::detail::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>+00024034
[ 27] 0x00007fff76d42ca6                            bin\win64\pgo\m_lxe.dll+01846438 mwboost::archive::detail::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>::oserializer<mwboost::archive::binaryTerm_oarchive,MathWorks::lxe::ClosedAbstractionFunctionDescriptorOwningIrTree>+00023902
[ 28] 0x00007fff8798d37a                    bin\win64\pgo\m_interpreter.dll+00054138 inEvalCmdWithLocalReturnInDesiredWSAndPublishEvents+00000074
[ 29] 0x00007fff86ff5bbf   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00744383 iqm::InternalEvalPlugin::inEvalCmdWithLocalReturn+00000111
[ 30] 0x00007fff86ff553e   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00742718 iqm::InternalEvalPlugin::execute+00000254
[ 31] 0x00007fff851c027e   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00524926 services::lmgr::exception::LicensingStartupException::~LicensingStartupException+00009486
[ 32] 0x00007fff86fd7a3f   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00621119 iqm::Iqm::setupIqmFcnPtrs+00113615
[ 33] 0x00007fff86fac9de   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00444894 iqm::Iqm::create+00030302
[ 34] 0x00007fff86f51d1d   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00072989 iqm::BaseIqmPlugin::processMFileError+00001357
[ 35] 0x00007fff85e0765b C:\Program Files\MATLAB\R2020b\bin\win64\mlutil.dll+03176027 cmddistributor::IIPRunNowMessage::deliver+00000027
[ 36] 0x00007fff8b489fc7 C:\Program Files\MATLAB\R2020b\bin\win64\libmwms.dll+01744839 foundation::msg_svc::exchange::MessageQueue::deliver+00000215
[ 37] 0x00007fff8b48bb35 C:\Program Files\MATLAB\R2020b\bin\win64\libmwms.dll+01751861 foundation::msg_svc::exchange::MessageQueue::sendReply+00004997
[ 38] 0x00007fff8b471c31 C:\Program Files\MATLAB\R2020b\bin\win64\libmwms.dll+01645617 foundation::msg_svc::exchange::CommunicationErrorEvent::getTransportID+00087009
[ 39] 0x00007fff8b47f079 C:\Program Files\MATLAB\R2020b\bin\win64\libmwms.dll+01699961 foundation::msg_svc::exchange::CommunicationErrorEvent::getTransportID+00141353
[ 40] 0x00007fff8b47dc02 C:\Program Files\MATLAB\R2020b\bin\win64\libmwms.dll+01694722 foundation::msg_svc::exchange::CommunicationErrorEvent::getTransportID+00136114
[ 41] 0x00007fff85c7983a C:\Program Files\MATLAB\R2020b\bin\win64\mlutil.dll+01546298 cmddistributor::IIP::EventMgrBypass::operator=+00050906
[ 42] 0x00007fff85c4dbd7 C:\Program Files\MATLAB\R2020b\bin\win64\mlutil.dll+01366999 cmddistributor::DebugLoopEvent::serializeExplicit+00021463
[ 43] 0x00007fffa68baec6   C:\Program Files\MATLAB\R2020b\bin\win64\mvm.dll+00437958 mvm::MVM::setClientCapabilities+00008902
[ 44] 0x00007fff851ddfca   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00647114 mcr::runtime::jmi_proxy::setJmiInitOptions+00090410
[ 45] 0x00007fff8515e7ad   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00124845 mcrFunctionSignature::set_signature+00103437
[ 46] 0x00007fff8517baea   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00244458 mcrFunctionSignature::set_signature+00223050
[ 47] 0x00007fff87004504   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00804100 iqm::PackagedTaskPlugin::execute+00000084
[ 48] 0x00007fff851c03ce   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00525262 services::lmgr::exception::LicensingStartupException::~LicensingStartupException+00009822
[ 49] 0x00007fff86fd7a3f   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00621119 iqm::Iqm::setupIqmFcnPtrs+00113615
[ 50] 0x00007fff86fa6b4a   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00420682 iqm::Iqm::create+00006090
[ 51] 0x00007fffa0b57a3e C:\Program Files\MATLAB\R2020b\bin\win64\libmwbridge.dll+00096830 ioReadLine+00000478
[ 52] 0x00007fffa0b5781c C:\Program Files\MATLAB\R2020b\bin\win64\libmwbridge.dll+00096284 ioReadLine+00000188
[ 53] 0x00007fffa0b6ba50 C:\Program Files\MATLAB\R2020b\bin\win64\libmwbridge.dll+00178768 mnGetCommandLineBuffer+00000304
[ 54] 0x00007fffa0b6beda C:\Program Files\MATLAB\R2020b\bin\win64\libmwbridge.dll+00179930 mnParser+00000506
[ 55] 0x00007fff851c62f4   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00549620 mcr_set_enableReadingFromStdin+00016084
[ 56] 0x00007fff8515e7ad   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00124845 mcrFunctionSignature::set_signature+00103437
[ 57] 0x00007fff8517baea   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00244458 mcrFunctionSignature::set_signature+00223050
[ 58] 0x00007fff87004504   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00804100 iqm::PackagedTaskPlugin::execute+00000084
[ 59] 0x00007fff851c03ce   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00525262 services::lmgr::exception::LicensingStartupException::~LicensingStartupException+00009822
[ 60] 0x00007fff86fd7a3f   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00621119 iqm::Iqm::setupIqmFcnPtrs+00113615
[ 61] 0x00007fff86fa7c42   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00425026 iqm::Iqm::create+00010434
[ 62] 0x00007fff86fa7421   C:\Program Files\MATLAB\R2020b\bin\win64\iqm.dll+00422945 iqm::Iqm::create+00008353
[ 63] 0x00007fff851960dc   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00352476 mcrInstantiationError::operator=+00011756
[ 64] 0x00007fff85196a3c   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00354876 mcrInstantiationError::operator=+00014156
[ 65] 0x00007fff851943cc   C:\Program Files\MATLAB\R2020b\bin\win64\mcr.dll+00345036 mcrInstantiationError::operator=+00004316
[ 66] 0x00007fffb4b4a2a3 C:\Program Files\MATLAB\R2020b\bin\win64\mwboost_thread-vc141-mt-x64-1_70.dll+00041635 mwboost::thread::swap+00000083
[ 67] 0x00007fffc221268a                   C:\windows\System32\ucrtbase.dll+00140938 o_exp+00000090
[ 68] 0x00007fffc3c77974                   C:\windows\System32\KERNEL32.DLL+00096628 BaseThreadInitThunk+00000020
[ 69] 0x00007fffc5ffa0b1                      C:\windows\SYSTEM32\ntdll.dll+00368817 RtlUserThreadStart+00000033


This error was detected while a MEX-file was running. If the MEX-file
is not an official MathWorks function, please examine its source code
for errors. Please consult the External Interfaces Guide for information
on debugging MEX-files.

```
'''

**Comment by kilir** [comment]
Please try also with an empty extern/nfft_openMP/*

**Comment by eddahbi-fis** [comment]
Thanks @kilir . Emptied both folders and now it works perfectly!

**Comment by ralfHielscher** [comment]
We just released MTEX 5.5.2 which addresses this issue. 
Could you please test it.

Ralf.
